src/webcontents-handler.ts

```diff
@@ -165,7 +166,8 @@ function onLinkContextMenu(ev: Event, params: ContextMenuParams, webContents: We
                             const resp = await fetch(url);
                             if (!resp.ok) throw new Error(`unexpected response ${resp.statusText}`);
                             if (!resp.body) throw new Error(`unexpected response has no body ${resp.statusText}`);
-                            pipeline(resp.body, fs.createWriteStream(filePath));
+                            const promisfyPipeline = promisify(pipeline);
+                            promisfyPipeline(resp.body, fs.createWriteStream(filePath));
```

Comment:
Should we not `await` the returned promise, to ensure that any errors are handled?

Reply:
Also: the `stream` library provides a promise-returning variant in `"stream/promises"` (see https://nodejs.org/api/stream.html#streams-promises-api). Probably better to use that than a separate `promisify`.

Reply:
@richvdh 
> Should we not await the returned promise, to ensure that any errors are handled?
Yes, right.

> Also: the stream library provides a promise-returning variant in "stream/promises" (see https://nodejs.org/api/stream.html#streams-promises-api). Probably better to use that than a separate promisify.

It was a better solution. Thanks :)